src/library/field.h

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Move `library` under `bench/binary` or `tests/binary` since it's only used in tests, probably also rename to something like binary_helper

Reply:
https://github.com/BlazingDB/libgdf/tree/binary-operators-draft/src/tests/binary-operation/util